CharacterCreation/06_Skills.txt

```diff
@@ -90,9 +94,9 @@ are listed below.
 Intimidation:	To strike fear into the hearts of your enemies
 	
 == Intelligence ==
-Hacking:	To pwn computer systems
-Medicine:	To repair life forms
-Repair:	To repair machines
+Hacking [I]:	To pwn computer systems
+Medicine [I]:	To heal biological life forms and biotic machines
+Repair [I]:	To repair mechanical life forms and abiotic machines

```

Comment:
Given the (very) tight lvl 1 to early-game economy (few classes can afford $250 for a trauma kit; they can barely afford repair and med kits as it is) this effectively nerfs most character's healing rolls by ~20. Repair and Medicine DCs are already quite high even taking the consumable kits into account; we need to compensate unless we're making the game more lethal on purpose.

Reply:
I have been intending to give out a toolkit at level one from a selection based on class, which would help. At least Soldier and Hunter (in addition to doc/engi) would probably have healing kits in the selection.

But it is a relevant point that, with disadvantage, a character would fail the DC 35 check to use a medikit/repair kit about half the time (with no skill ranks and +0 attr mod), which is probably a feel bad. As technology advances so does soldier-proofing (usually), so I wonder if we really need to have that check?

Reply:
@bleehu

Reply:
Tools given